TWCManager.py

```diff
@@ -115,13 +114,17 @@
 
 
 def debugLog(minlevel, message):
-    if config["config"]["debugLevel"] >= minlevel:
-        print(
-            colored(master.time_now() + " ", "yellow")
-            + colored("TWCManager", "green")
-            + colored(f(" {minlevel} "), "cyan")
-            + f("{message}")
-        )
+    if master == None:
+        # I don't think any call will come through this
+        if config["config"]["debugLevel"] >= minlevel:
+            print(
+                colored(master.time_now() + " ", "yellow")
+                + colored("TWCManager", "green")
+                + colored(f(" {minlevel} "), "cyan")
+                + f("{message}")
+            )
+    else:
+        master.debugLog(minlevel, "TWCManager", message)
```

Comment:
We can test if this is the case, but I don't think leaving both paths makes sense, it doesn't deprecate the legacy function but does potentially add complexity. I understand this is for the debug logging, and I agree we need to unify, but to my mind resolving the question of whether a local function is required is the first step and is pretty easy to test.

Reply:
I agree. Just did not want do break any code...

Reply:
No problems, it serves as a prompt for this to be tested. I won't be able to just yet but will test this when I get a chance, but you're also welcome to take it for a spin and see if it ever executes the code branch perhaps by altering the output for that branch to flag this